spec/helpers/teams_helper_spec.rb

```diff
@@ -8,11 +8,11 @@
     let(:team_with) { conference_preference.team }
 
     it 'should return false if don\'t exists conference persisted' do
```

Comment:
Escaping within strings always looks kinda weird if there's no "real" reason for it 😉 
I'd suggest to either use double quotes here, or to just rename the example to something like
> it 'returns false if no conference_preference was persisted for the team'

Combining "it" and "should" for an RSpec example is also a bit weird, since they are actually both keywords for the same...